__tests__/app.test.js

```diff
@@ -154,3 +153,72 @@ describe('GET /api/reviews/:review_id/comments', () => {
   });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+  const newComment = {
+    username: 'weegembump',
+    body: 'Not Enough DICE!!!!'
+  }
+  test('should add a comment to the database and respond wiht the newly created comment', () => {
+    return request(app)
+    .post("/api/reviews/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then(({ body }) => {
+      const { amendedComment } = body;
+      const editedComment = [{
+        comment_id: expect.any(Number),
+        body: 'Not Enough DICE!!!!',
+        review_id: 1,
+        author: 'weegembump',
+        votes: 0,
+        created_at: expect.any(String)
+      }]
+      expect(amendedComment).toEqual(editedComment);
```

Comment:
I think the variable names are a little confusing here - "amended" and "edited" sort of mean the same thing?
Would still recommend using `expect.objectContaining()` rather than just `.toEqual` alone.
Preferably, the comment on the response body shouldn't be an array containing a single comment, just that comment